### DISCUSSION TITLE
I'd really love a good tutorial for people new to identity management

### DISCUSSION THREAD

**Discussion by jike212** [goauthentik/authentik#1461] [Q&A]
I've been looking for something that can handle authentication, SSL proxying to internal services, and LDAP and Authentik seems to be the best choice.

The problem is I'm still a bit new to this world and can't seem to successfully test proxying with authentication a local dev server I have on my computer for testing.

For example, I've gone through the docker-compose setup and added my internal test application (an HTTP server running on port 3040 on my local computer) as well as a proxy provider. I've also defined a local host name in my hosts file (e.g. `proxy.myhost.lan` which resolves to `127.0.0.1`)

When creating the proxy provider for the application, I set the following settings:
* `External Host` - `http://proxy.myhost.lan:3050` (this is just to test to make sure the proxying is working properly
* `Internal Host` - `http://proxy.myhost.lan:3040`

When I attempt to go to `http://proxy.myhost.lan:3050` which is what I set for the external host, the browser fails to connect and the Authentik logs do not display any connection attempt.

**Comment by BeryJu** [comment]
Hi, first off, I totally agree with the tutorials, however I have a full time job and only have so much time, so I can't do much there.

For the issue you're having, change the `External Host` to something ending in port 9000, as thats where authentik listens by default.

**Reply by jike212** [reply]
Oh, sorry I didn't mean just you, you've put quite a lot of time into this already (and thank you for that!)

I did what you suggested and changed `External Host` to `http://proxy.myhost.lan:9000` (kept internal host the same and listening on port 3040) and I do see a login prompt for Authentik (not sure if its for logging into Authentik itself or into my internal service)

However, when I enter `akadmin` into the username prompt, and press `Log In`, the page simply refreshes and keeps me on the login page (with nowhere to enter a password or continue through the process) and the following two entries in the log appear:

```
{
   "event":"/api/v3/flows/executor/default-authentication-flow/?query=next%3D%252F",
   "host":"172.18.0.1",
   "level":"info",
   "location":"/api/v3/flows/executor/default-authentication-flow/?query=next%3D%252F",
   "logger":"authentik.asgi",
   "method":"POST",
   "pid":26,
   "request_id":"1b4bcf423f7d41a79e03062e9394cee3",
   "runtime":0,
   "scheme":"http",
   "size":0.031,
   "status":302,
   "timestamp":"2021-09-25T04:24:35.372006",
   "user_agent":"<removed>"
},{
   "event":"/api/v3/flows/executor/default-authentication-flow/?query=next%3D%252F",
   "host":"172.18.0.1",
   "level":"info",
   "logger":"authentik.asgi",
   "method":"GET",
   "pid":26,
   "request_id":"e9c50958d36d4bf7a372ba91801eb6a2",
   "runtime":312,
   "scheme":"http",
   "size":0.036,
   "status":200,
   "timestamp":"2021-09-25T04:24:35.423566",
   "user_agent":"<removed>"
}
```
